tomni/cytosmart_data_format/annotations/polygon/main.py

```diff
@@ -101,6 +109,38 @@ def perimeter(self) -> float:
     def points(self) -> List[Point]:
         return self._points
 
+    @points.setter
+    def points(self, value: List[Point]) -> None:
+        """
+        Simplify every list of points to only have points that add information
```

Comment:
does it remove more points if you load a annotation multiple times? e.g cdf.from_dict(cdf.to_dict())

Reply:
No, because all the points that are removed the first time are points without information.
If you load it a second time there are no points without information to remove

Reply:
can you add this as unittest